### PR TITLE
[BULK] DocuTune - Fix build validation issues: docs-link-absolute (part 8)

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

**Comment by brittmsantos** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@sdwheeler

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

File: docset/winserver2022-ps/international/Set-WinDefaultInputMethodOverride.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## INPUTS
 
 ### InputTIP
-You can pipe a locale ID and a keyboard language ID (KLID). See [Default Input Profiles (Input Locales) in Windows](https://docs.microsoft.com/windows-hardware/manufacture/desktop/default-input-locales-for-windows-language-packs)
+You can pipe a locale ID and a keyboard language ID (KLID). See [Default Input Profiles (Input Locales) in Windows](/windows-hardware/manufacture/desktop/default-input-locales-for-windows-language-packs)
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/international/Set-WinDefaultInputMethodOverride.md:59]:
```suggestion
You can pipe a locale ID and a keyboard language ID (KLID). See [Default Input Profiles (Input Locales) in Windows](/windows-hardware/manufacture/desktop/default-input-locales-for-windows-language-packs).
```

File: docset/winserver2022-ps/storage/Get-VirtualDisk.md [hunk 2]
```diff
@@ -624,7 +624,7 @@ You can use the pipeline operator to pass an MSFT_TargetVirtualDisk object to th
 ## OUTPUTS
 
 ### Microsoft.Management.Infrastructure.CimInstance#ROOT/Microsoft/Windows/Storage/MSFT_VirtualDisk
-This cmdlet outputs an object that represents the specified virtual disk. For more information about output object type, please see [MSFT_VirtualDisk class](https://docs.microsoft.com/previous-versions/windows/desktop/stormgmt/msft-virtualdisk)
+This cmdlet outputs an object that represents the specified virtual disk. For more information about output object type, please see [MSFT_VirtualDisk class](/previous-versions/windows/desktop/stormgmt/msft-virtualdisk)
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/storage/Get-VirtualDisk.md:627]:
```suggestion
This cmdlet outputs an object that represents the specified virtual disk. For more information about output object type, see [MSFT_VirtualDisk class](/previous-versions/windows/desktop/stormgmt/msft-virtualdisk).
```

File: docset/winserver2022-ps/tls/Get-TlsCipherSuite.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ The **Get-TlsCipherSuite** cmdlet gets an ordered collection of cipher suites fo
 
 For more information about the TLS cipher suites, see the documentation for the Enable-TlsCipherSuite cmdlet or type `Get-Help Enable-TlsCipherSuite`.
 
-For more information about protocol versions , see [BCRYPT_KDF_TLS_PRF (L"TLS_PRF")](https://docs.microsoft.com/windows/desktop/api/bcrypt/nf-bcrypt-bcryptderivekey#bcrypt_kdf_tls_prf-ltls_prf) .
+For more information about protocol versions , see [BCRYPT_KDF_TLS_PRF (L"TLS_PRF")](/windows/desktop/api/bcrypt/nf-bcrypt-bcryptderivekey#bcrypt_kdf_tls_prf-ltls_prf) .
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/tls/Get-TlsCipherSuite.md:27]:
```suggestion
For more information about protocol versions , see [BCRYPT_KDF_TLS_PRF (L"TLS_PRF")](/windows/desktop/api/bcrypt/nf-bcrypt-bcryptderivekey#bcrypt_kdf_tls_prf-ltls_prf).
```

File: docset/winserver2022-ps/windowserrorreporting/Disable-WindowsErrorReporting.md [hunk 4]
```diff
@@ -27,7 +27,7 @@ Windows Error Reporting generates reports in response to system events, such as
 
 To get the current WER status, use the [Get-WindowsErrorReporting](./Get-WindowsErrorReporting.md) cmdlet.
 If you disable WER, you can use the [Enable-WindowsErrorReporting](./Enable-WindowsErrorReporting.md) cmdlet to re-enable it.
-After you run this cmdlet, WER again sends information about application failures to Microsoft. To customize the settings of Windows Error Reporting, use registry keys described in [WER Settings](https://docs.microsoft.com/windows/desktop/wer/wer-settings) article.
+After you run this cmdlet, WER again sends information about application failures to Microsoft. To customize the settings of Windows Error Reporting, use registry keys described in [WER Settings](/windows/desktop/wer/wer-settings) article.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/windowserrorreporting/Disable-WindowsErrorReporting.md:30]:
```suggestion
After you run this cmdlet, WER again sends information about application failures to Microsoft. To customize the settings of Windows Error Reporting, use registry keys described in the [WER Settings](/windows/desktop/wer/wer-settings) article.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @alexbuckgit, a few minor additional changes, please.

File: docset/winserver2022-ps/rdmgmt/Set-RDCertificate.md [hunk 5]
```diff
@@ -178,7 +178,7 @@ Accept wildcard characters: False
 ### CommonParameters
 This cmdlet supports the following common parameters: *-Debug*, *-ErrorAction*, *-ErrorVariable*, *-InformationAction*, *-InformationVariable*, *-OutVariable*, *-OutBuffer*, *-PipelineVariable*, *-Verbose*, *-WarningAction*, and *-WarningVariable*.
 
-For more information, see [about_CommonParameters](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_commonparameters?view=powershell-6).
+For more information, see [about_CommonParameters](/powershell/module/microsoft.powershell.core/about/about_commonparameters?view=powershell-6).
```
Comment (sdwheeler) [inline docset/winserver2022-ps/rdmgmt/Set-RDCertificate.md:181]:
```suggestion
For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/rdmgmt/Set-RDCertificate.md [hunk 6]
```diff
@@ -196,4 +196,4 @@ For more information, see [about_CommonParameters](https://docs.microsoft.com/po
 
 [New-RDCertificate](./New-RDCertificate.md)
 
-[about_CommonParameters](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_commonparameters?view=powershell-6)
+[about_CommonParameters](/powershell/module/microsoft.powershell.core/about/about_commonparameters?view=powershell-6)
```
Comment (sdwheeler) [inline docset/winserver2022-ps/rdmgmt/Set-RDCertificate.md:199]:
```suggestion
[about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216)
```

File: docset/winserver2022-ps/scheduledtasks/Register-ScheduledTask.md [hunk 7]
```diff
@@ -342,7 +342,7 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_commonparameters).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](/powershell/module/microsoft.powershell.core/about/about_commonparameters).
```
Comment (sdwheeler) [inline docset/winserver2022-ps/scheduledtasks/Register-ScheduledTask.md:345]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]

**Comment by alexbuckgit** [comment]
@JohanFreelancer9 @sdwheeler -

The request changes have been accepted. Please merge, thanks!

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**